ibis/backends/tests/test_asof_join.py

```diff
@@ -84,6 +84,7 @@ def time_keyed_right(time_keyed_df2):
 )
 @pytest.mark.notyet(
     [
+        "clickhouse",
```

Comment:
is this because it also truncates to seconds, as in the test below? Does this PR therefore regress behavior for clickhouse (eg a user could be relying on some behavior, and we break it here)? Did you think if there would be a way to avoid this regression? If it's too hard, I could be fine with the regression, but at least worth talking about trying to avoid a regression.